content/blogposts/2023/announcing-scip-clang.md

```diff
@@ -0,0 +1,84 @@
+for C and C++ code written from the ground up to natively emit [SCIP](https://github.com/sourcegraph/scip)
+and especially support the wide range of language features present in C++.
+
+scip-clang is now available in beta, and replaces our older indexer lsif-clang.
+We've successfully indexed large codebases such as Chromium with scip-clang.
+
+Here's a quick demo showcasing some features:
+
+{/* TODO: Add demo here */}
+
+scip-clang supports a superset of the functionality of lsif-clang.
+The main additions are:
+
+- scip-clang uses a multi-processing architecture instead of multi-threading,
+  so that indexing failures (such as crashes) in a single translation unit
+  do not affect indexing for other translation units.
+- scip-clang natively supports code navigation for `#include` pragmas and macros.
+- scip-clang is based on Clang 16 instead of Clang 11.
+  It consumes Clang as a library rather than as a fork,
+  making it easy to update the version of Clang used in the future. 
+
+Some other minor quality-of-life improvements include:
+
+- scip-clang infers paths to standard headers for GCC and Clang
+  from the compilation database without requiring extra command-line flags.
+- scip-clang binaries are available for both Linux and macOS.
+- Despite supporting more features, thanks to SCIP, index sizes
+  (both compressed and uncompressed) are about
+  10%-20% the size of the corresponding LSIF indexes.
+  This translates to faster uploads, a lower likelihood of upload errors,
+  and reduced risk of out-of-memory when the Sourcegraph backend
+  processes an index.
+- scip-clang uses incremental parsing for compilation databases,
+  reducing the risk of out-of-memory errors on ingestion.
+
+If you are a Sourcegraph customer using C or C++,
+please [try out scip-clang](https://github.com/sourcegraph/scip-clang),
+and let us know if you run into issues,
+or if you have feedback for improvement.
+
+As with our other indexers, open source maintainers are also welcome
+to use scip-clang to index their projects
+and upload indexes to Sourcegraph.com
+to benefit from precise code navigation for C and C++.
+
+## A word about performance
+
+scip-clang requires a traversal of the abstract syntax tree
+after type-checking the code, so that type information is available.
+
+Compared to a baseline of type-checking all translation units in parallel,
+scip-clang takes about 30% more time.
+
+In the future, we will be able to reduce this overhead in two ways:
+- [Optimizing the release build](https://github.com/sourcegraph/scip-clang/issues/27)
+  by using well-known techniques such as ThinLTO and PGO.
+- [Parallelizing the index merging step](https://github.com/sourcegraph/scip-clang/issues/139),
+  which combines information about forward declarations
+  and project-external symbols across translation units.
+  Depending on codebase size and core count,
+  about 10%-40% of the overall indexing time is spent in index merging.
```

Comment:
There are 3 different figures given in this performance section, 30% more time, 10-40%, 80-90%.  I understand it's for different stage but it's hard at a glance to come away with what perf to expect.  

Perhaps pulling that section into a table and then showing overall expected perf and then it runs into your rationale about how we're going to work on improving it etc.

Reply:
Create a table to show this in a better way. Can you take a second look?

Reply:
Looks better!